stable_baselines/common/vec_env/subproc_vec_env.py

```diff
@@ -87,7 +87,7 @@ def close(self):
 
     def render(self, mode='human', **kwargs):
         for pipe in self.remotes:
-            pipe.send(('render', kwargs))
+            pipe.send(('render', {'mode':'rgb_array', **kwargs}))
```

Comment:
why not `'mode': mode` ?

Reply:
oh, i see, it sends to the subprocess and then depending on mode, it will show or not the img
I think a comment here could be a good idea ;)